modules/eplanningBidAdapter.js

```diff
@@ -236,17 +252,42 @@ function getSpacesStruct(bids) {
   return e;
 }
 
+function getFirstSizeVast(sizes) {
+  if (sizes == undefined || !Array.isArray(sizes)) {
+    return undefined;
+  }
+
+  let size = Array.isArray(sizes[0]) ? sizes[0] : sizes;
+
+  return (Array.isArray(size) && size.length == 2) ? size : undefined;
+}
+
 function cleanName(name) {
   return name.replace(/_|\.|-|\//g, '').replace(/\)\(|\(|\)|:/g, '_').replace(/^_+|_+$/g, '');
 }
 
 function getSpaces(bidRequests, ml) {
+  let impType = bidRequests.reduce((previousBits, bid) => (bid.mediaTypes && bid.mediaTypes[VIDEO]) ? (bid.mediaTypes[VIDEO].context == 'outstream' ? (previousBits | 2) : (previousBits | 1)) : previousBits, 0);
```

Comment:
Not required, but for better readability, would you want to replace the part
`bid.mediaTypes[VIDEO].context == 'outstream' ? (previousBits | 2) : (previousBits | 1)`  

by 

`bid.mediaTypes[VIDEO].context == 'outstream' ? (previousBits | VAST_OUTSTREAM) : (previousBits | VAST_INSTREAM)`